src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -95,12 +97,15 @@ export default function EnvTriggerView() {
     const [selectedAppID, setSelectedAppID] = useState<number>()
     const [selectedAppList, setSelectedAppList] = useState<WorkflowAppSelectionType[]>([])
     const [workflows, setWorkflows] = useState<WorkflowType[]>([])
+    const [filteredWorkflows, setFilteredWorkflows] = useState<WorkflowType[]>([])
     const [selectedCDNode, setSelectedCDNode] = useState<WorkflowNodeSelectionType>(null)
     const [selectedCINode, setSelectedCINode] = useState<WorkflowNodeSelectionType>(null)
     const [filteredCIPipelines, setFilteredCIPipelines] = useState(null)
     const [bulkTriggerType, setBulkTriggerType] = useState<DeploymentNodeType>(null)
     const [materialType, setMaterialType] = useState(MATERIAL_TYPE.inputMaterialList)
     const [responseList, setResponseList] = useState<ResponseRowType[]>([])
+    const [isSelectAll, setSelectAll] = useState(false)
+    const [selectAllValue, setSelectAllValue] = useState<'CHECKED' | 'INTERMEDIATE'>('CHECKED')
```

Comment:
Can we use - existing CHECKBOX_VALUE enum type?

Reply:
Done

Reply:
Done